python/paddle/fluid/initializer.py

```diff
@@ -756,7 +756,7 @@ def __call__(self, var, block):
             values = [int(v) for v in self._value.flat]
         else:
             raise ValueError("Unsupported dtype %s", self._value.dtype)
-        if self._value.size > 1024 * 1024 * 5:
+        if self._value.size > 1024 * 1024 * 1024:
```

Comment:
why change this?

Reply:
NumpyArrayInitializer limits numpy size. I think a few GBs is OK